README.md

```diff
@@ -4,48 +4,108 @@
 
-<p align="center">
-  <a href="https://readthedocs.org/projects/tephi/">
-    <img src="https://readthedocs.org/projects/tephi/badge/?version=latest"
-         alt="Documentation Status" />
-  </a>
-  <a href="https://coveralls.io/github/SciTools/tephi?branch=main">
-    <img src="https://coveralls.io/repos/github/SciTools/tephi/badge.svg?branch=main"
-         alt="Coverage Status" />
-  </a>
-  <a href="https://github.com/psf/black">
-    <img src="https://img.shields.io/badge/code/style-black-000000.svg"
-         alt="Black" />
-  </a>
-  <a href="https://results.pre-commit.ci/latest/github/SciTools/tephi/main">
-    <img src="https://results.pre-commit.ci/badge/github/SciTools/tephi/main.svg"
-         alt="pre-commit.ci status">
-  </a>
-  <a href="https://mybinder.org/v2/gh/SciTools/tephi/main?filepath=index.ipynb">
-    <img src="https://mybinder.org/badge_logo.svg"
-         alt="Launch MyBinder" />
-  </a>
-</p>
+|         |                                                                                                                                                                                                                                                                                                                                   |
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CI      | [![Documentation Status](https://readthedocs.org/projects/tephi/badge/?version=latest)](https://readthedocs.org/projects/tephi/) [![Pre-Commit CI Status](https://results.pre-commit.ci/badge/github/SciTools/tephi/main.svg)](https://results.pre-commit.ci/latest/github/SciTools/tephi/main)                                   |
+| Health  | [![Coverage Status](https://coveralls.io/repos/github/SciTools/tephi/badge.svg?branch=main)](https://coveralls.io/github/SciTools/tephi?branch=main)                                                                                                                                                                              |
+| Meta    | [![Code style: Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)                                                                                                                                                                                                                  | 
+| Package | [![Launch MyBinder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/SciTools/tephi/main?filepath=index.ipynb) [![conda-forge](https://img.shields.io/conda/vn/conda-forge/tephi)](href="https://anaconda.org/conda-forge/tephi)  [![PyPI](https://img.shields.io/pypi/v/tephi)](https://pypi.org/project/tephi/) |
```

Comment:
The URL for the `conda-forge` badge isn't correct.

Seems like a cut'n paste blip i.e., 

`(href="https://anaconda.org/conda-forge/tephi)` -> `(https://anaconda.org/conda-forge/tephi)`